internal/testdata/applications/app3.desktop

```diff
@@ -1,4 +1,4 @@
 [Desktop Entry]
 Name=App3
 Exec=app3
-Icon=app3
+Icon=/home/stephen/go/src/fyne.io/desktop/internal/testdata/icons/app3.png
```

Comment:
This looks like a mistaken commit - we need to fix this being overwritten all the time

Reply:
I don't think it breaks the mac files? Why would it? Also yes. Need to add that file to .gitignore

Reply:
Quite right, apologies I got confused between the two changes.
Strip this line and we're good